docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = source
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
```

Comment:
When I call this make file, I have to run `poetry run make ...`, As I'm using poetry with it's default setting `virtualenvs.create True` and no venv.

the fact your git ignoring `.venv` makes me thing you must be using poetry with `virtualenvs.create False` and within a venv shell. 

In Gabe's [makefile](https://github.com/NREL/BuildingMOTIF/pull/1/files) and the current README, commands include `poetry run`, assuming the default, non-venv configuration. We can have a dicussion about what we want to do, but ultimately, the docs and entrypoints should uniformly choose one or the other.

Reply:
Sounds good. We were using poetry _with_ a virtual environment in tasty, so that's what I did here. I don't have a preference either way, however no venv seems cleaner/simpler.

Reply:
from the team meeting yesterday we decided that without venv is cleaner and also that these kind of scripts should not assume the user is already in the virtual environment and as such should use `poetry run <cmd>`. This has the added benefit that it should work in either case (in virtual env or not).